### ISSUE TITLE
[Bug Report] [Component] [upload] 上传组件 list-type=picture-card且show-file-list=false时，不显示上传组件

### ISSUE THREAD

**Issue by aliang888** [element-plus/element-plus#7674]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.33`
- Element Plus Version: `2.2.0`
- Browser / OS: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36`
- Build Tool: `Vite`

### Related Component

- `el-upload`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtdXBsb2FkXG4gICAgY2xhc3M9XCJ1cGxvYWQtZGVtb1wiXG4gICAgZHJhZ1xuICAgIGFjdGlvbj1cImh0dHBzOi8vanNvbnBsYWNlaG9sZGVyLnR5cGljb2RlLmNvbS9wb3N0cy9cIlxuICAgIGxpc3QtdHlwZT1cInBpY3R1cmUtY2FyZFwiXG4gICAgOnNob3ctZmlsZS1saXN0PWZhbHNlXG4gICAgICAgICAgICAgXG4gID5cbiAgICA8ZWwtaWNvbiBjbGFzcz1cImVsLWljb24tLXVwbG9hZFwiPjx1cGxvYWQtZmlsbGVkIC8+PC9lbC1pY29uPlxuICAgIDxkaXYgY2xhc3M9XCJlbC11cGxvYWRfX3RleHRcIj5cbiAgICAgIERyb3AgZmlsZSBoZXJlIG9yIDxlbT5jbGljayB0byB1cGxvYWQ8L2VtPlxuICAgIDwvZGl2PlxuICAgIDx0ZW1wbGF0ZSAjdGlwPlxuICAgICAgPGRpdiBjbGFzcz1cImVsLXVwbG9hZF9fdGlwXCI+XG4gICAgICAgIOS4iumdoueci+S4jeWIsOS4iuS8oOe7hOS7tjxicj5cbiAgICAgICAganBnL3BuZyBmaWxlcyB3aXRoIGEgc2l6ZSBsZXNzIHRoYW4gNTAwa2JcbiAgICAgIDwvZGl2PlxuICAgIDwvdGVtcGxhdGU+XG4gIDwvZWwtdXBsb2FkPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IFVwbG9hZEZpbGxlZCB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

### Steps to reproduce

```vue
<template>
  <el-upload
    class="upload-demo"
    drag
    action="https://jsonplaceholder.typicode.com/posts/"
    list-type="picture-card"
    :show-file-list=false
             
  >
    <el-icon class="el-icon--upload"><upload-filled /></el-icon>
    <div class="el-upload__text">
      Drop file here or <em>click to upload</em>
    </div>
    <template #tip>
      <div class="el-upload__tip">
        上面看不到上传组件<br>
        jpg/png files with a size less than 500kb
      </div>
    </template>
  </el-upload>
</template>

<script setup lang="ts">
import { UploadFilled } from '@element-plus/icons-vue'
</script>

```

### What is Expected?

显示上传组件

### What is actually happening?

不显示上传组件

### Additional comments

_(empty)_

![upload](https://user-images.githubusercontent.com/19453262/168203827-30ab0372-3092-4a78-b986-2651b9959621.jpg)

**Comment by chenxch** [comment]
picture-card目前是作为照片墙使用的，你可以尝试其他模式搭配trigger插槽 
```html
<template #trigger>
  <div class="el-upload el-upload--picture-card" tabindex="0">
    <el-icon><Plus class="el-icon" /></el-icon>
    <input class="el-upload__input" name="file" accept="" type="file"></div>
</template>
```

**Comment by weiguanglai** [comment]
> picture-card目前是作为照片墙使用的，你可以尝试其他模式搭配trigger插槽
> 
> ```
> <template #trigger>
>   <div class="el-upload el-upload--picture-card" tabindex="0">
>     <el-icon><Plus class="el-icon" /></el-icon>
>     <input class="el-upload__input" name="file" accept="" type="file"></div>
> </template>
> ```

PR;9607有人提交了，可否合并，项目中也遇到了这种情况

**Comment by HeftyKoo** [comment]
fixed by #9607
